test/client/fixtures/sandbox/console-test.js

```diff
@@ -0,0 +1,66 @@
+var consoleSandbox = hammerhead.sandbox.console;
+
+function argsToArray (args) {
+    var res = [];
+    var i   = 0;
+
+    while (i < args.length) {
+        res.push(args[i]);
+        i++;
+    }
+
+    return res;
+}
+
+test('consoleMethCalled event', function () {
+    var log           = '';
+    var handledEvents = [];
+    var logFromEvents = '';
+
+    /* eslint-disable no-console */
+    var originMethods = {
+        log:   console.log,
+        warn:  console.warn,
+        error: console.error,
+        info:  console.info
+    };
+
+    function addToLog (meth, args) {
+        log += argsToArray(args).join('');
+
+        originMethods[meth].apply(console, args);
+    }
+
+    window.console = {
```

Comment:
To capture console trace you override methods that already overriden in sandbox.
Usually, we use an another way - we override the appropriate native methods.
See tests in https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/fixtures/sandbox/node/methods-test.js
Use this way too.